app/Notifications/Training/WaitingListRemovalCancelled.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Notifications\Training;
+
+use App\Notifications\Notification;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class WaitingListRemovalCancelled extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    private string $list_name;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct(string $list_name)
+    {
+        parent::__construct();
+
+        $this->list_name = $list_name;
+    }
```

Comment:
```suggestion
    /**
     * Create a new notification instance.
     *
     * @return void
     */
    public function __construct(private string $list_name)  { parent::__construct(); }
```

If you need to do no further work to the inbound properties of the constructor, consider using PHP8's property promotion: https://stitcher.io/blog/constructor-promotion-in-php-8

Reply:
I'd extend this to any class created in this PR where the constructor mainly just assigns some class level variables.